tests/e2e/features/artifacts/bridgePage/artifacts.feature

```diff
@@ -4,6 +4,24 @@ Feature: Withdraw
   Background:
     Given Connect Metamask extension with login action
 
+  @id1609 @id1607
+  Scenario: Check Explore ecosystem button on Bridge Deposit
+    Given I go to page "/bridge?network=era-goerli"
+    When I click by text "Deposit"
+    When I choose "ETH" as token and insert "0.0000000001" as amount
+    When I "confirm" transaction after clicking "Add funds to zkSync Era Testnet" button
+    Then Message "Transaction submitted" should be visible
+    Then Element with "text" " Explore ecosystem " should be "visible"
+    Then Element with "href" "https://ecosystem.zksync.io" should be "visible"
+        #id1607 Check other elements on window
+    Then Element with "partial class" "progress-plane-animation" should be "visible"
```

Comment:
please add steps from 1.3 from https://allure.matterlabs.dev/project/17/test-cases/1607?treeId=30

Reply:
Please carefully double check all steps, I see duplicates, is this expected? and also not sure about where is 1.3.2 and 1.3.7  
![image](https://github.com/matter-labs/dapp-portal/assets/51121511/21fefc16-d970-4125-8702-b6fcfac0b80e)